src/Helpers/CVEHelper.js

```diff
@@ -15,28 +15,17 @@ import messages from '../Messages';
  * @param {Array} expandedRows id of opened(expaned) row
  *
  */
```

Comment:
Can you please update/remove the outdated jsdoc comment above? I'd much rather keep it and update it and make it some rule to have at least small comment above new functions we add.